src/app/components/map/popup-box/popup-box.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="popup">Click me!
+<div class="popup">
 
-  <button (click)="click()" id="try">click</button>
+  <button (click)="click()" id="try">Zoom in</button>
   <span class="popuptext" id="myPopup">Popup text...</span>
```

Comment:
what is this for?

Reply:
Will delete the id, the popup text is for adding text in the popup box later